frontend/src/lib-components/messages/MessageEditor.tsx

```diff
@@ -420,20 +420,27 @@ export default React.memo(function MessageEditor({
         />
       </>
     )
+  const FlagsInfoContent = () => {
```

Comment:
You should not define a React component inside another component. React will see it as a new component every time and will not be able to reconcile it optimally. You should move the component to the top level, pass the `message` as a prop, and use `useTranslation()` to get `i18n`.

Reply:
Moved outside!